android/src/main/java/io/radar/react/RNRadarModule.java

```diff
@@ -527,6 +528,17 @@ public void setForegroundServiceOptions(ReadableMap optionsMap) {
         }
     }
 
+    @ReactMethod
+    public void setAndroidNotificationsOptions(ReadableMap optionsMap) {
```

Comment:
make this singular

also probably should make this just be `setNotificationOptions` — we might eventually have an iOS one and it'd be nice to make them the same (will probably need to be clever in how we handle some params)